### PR TITLE
Upgraded to ROS Noetic version.

### PR DESCRIPTION
**Pull request by Kokjix**
When i try to building ROS Noetic on Ubuntu 20.04 I encountered building problems so i fixed the pointer issues.

### CODE REVIEW

**Comment by peci1** [comment]
Thank you for the PR. I'll have a look at it in a few days, and if it works, I'll trigger a Noetic release.

**Comment by peci1** [comment]
Thank you @Kokjix for working on this PR, it helped me get this working. However, I took a different approach utilizing the change from https://github.com/ros/robot_state_publisher/pull/115 (which I made some time ago with this package in mind).

Could you please test that it works for you? I added a unit test to this package to make sure it works on both Melodic and Noetic, but I'd like to have another confirmation before I release the new version.

I'm closing this PR in favor of #2.

File: cfg/DynamicRobotState.cfg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (peci1) [inline cfg/DynamicRobotState.cfg:1]: This is not needed (and possibly even wrong in ROS1). catkin automatically transforms shebangs to either python or python3 based on the python version you are building against (check the generated files in devel or install directories).

**Comment by Kokjix** [comment]
Thank you for review, I didn't know that. I will test branch in a few days, nowadays I have some internet issues

**Comment by Kokjix** [comment]
Hi @peci1, sorry to bother you I have a question. About testing:

>  Could you please test that it works for you? I added a unit test to this package to make sure it works on both Melodic and 

> Noetic, but I'd like to have another confirmation before I release the new version.

how should i test package?

**Comment by peci1** [comment]
`git clone https://github.com/peci1/dynamic_robot_state_publisher -b noetic-upgrade` in your workspace, then just catkin make the workspace, re-source setup.bash and you're running the updated version.

**Comment by Kokjix** [comment]
Hello again @peci1, I tested updated version of package on Han's robotics Elfin5 robot URDF and package was worked very well. Thank you for your work.

**Comment by peci1** [comment]
Thank you for testing the fix. I've merged into master and I'll trigger a Noetic binary release in a while. This means the APT-installable version should be available in Noetic in 3-4 weeks (or in a few hours/days on the ros-shadow-fixed repo).

**Pull request closed without merging**